content/homepage.js

```diff
@@ -184,12 +184,184 @@ const tools = {
+                    "alt": "element84 logo",
+                },
+            ],
+
+            [
+                {
+                    "class": w48,
+                    "site": "https://sparkgeo.com/",
+                    "image": "sparkgeo.png",
+                    "alt": "sparkgeo logo",
+                },
+                {
+                    "class": w48,
+                    "site": "https://www.maxar.com/",
+                    "image": "maxar-technologies.png",
+                    "alt": "maxar technologies logo",
+                },
+                {
+                    "class": w48,
+                    "site": "https://arturo.ai/",
+                    "image": "arturo.jpeg",
+                    "alt": "Arturo logo",
+                },
+                {
+                    "class": w48,
+                    "site": "https://www.usgs.gov/",
+                    "image": "usgs.png",
+                    "alt": "usgs logo",
+                },
+                {
+                    "class": w48,
+                    "site": "https://www.cosmiqworks.org/",
+                    "image": "cosmiqworks.png",
+                    "alt": "cosmiqworks logo",
+                },
+            ],
+
+            [
+                {
+                    "class": "md:w-32 m-3 p-2 md:p-4",
+                    "site": "https://aws.amazon.com/",
+                    "image": "aws.png",
+                    "alt": "aws logo",
+                },
+                {
+                    "class": w32,
+                    "site": "https://www.digitalearthafrica.org/",
+                    "image": "digital-earth-africa.png",
+                    "alt": "digital earth africa logo",
+                },
+                {
+                    "class": w32,
+                    "site": "https://www.worldbank.org/",
+                    "image": "world-bank-group.png",
+                    "alt": "world bank group logo",
+                },
+                {
+                    "class": w32,
+                    "site": "https://www.azavea.com/",
+                    "image": "azavea.png",
+                    "alt": "azavea logo",
+                },
+                {
+                    "class": w32,
+                    "site": "https://www.nasa.gov/",
+                    "image": "nasa.png",
+                    "alt": "NASA logo",
+                },
+            ],
+
+            [
+                {
+                    "class": w32,
+                    "site": "https://umbra.space/",
+                    "image": "umbra.png",
+                    "alt": "Umbra logo",
+                },
+                {
+                    "class": w32,
+                    "site": "https://openeo.org/",
+                    "image": "open-eo.png",
+                    "alt": "Open EO logo",
+                },
+                {
+                    "class": w32,
+                    "site": "", // what is this company? could find their site
```

Comment:
Attention pls.

Reply:
Ah, they're defunct - got bought by snap (and then half the original team just got laid off in their last round). I'd say leave the logo but it's fine if there's no link.